pkg/api/app_routes.go

```diff
@@ -18,8 +18,7 @@ import (
 
 var pluginProxyTransport = &http.Transport{
 	TLSClientConfig: &tls.Config{
-		InsecureSkipVerify: true,
-		Renegotiation:      tls.RenegotiateFreelyAsClient,
+		Renegotiation: tls.RenegotiateFreelyAsClient,
```

Comment:
This have to be configurable. In case people want to turn it off.

Reply:
Not sure where this should be configured. I guess we should make a new group 
```
[plugins] 
app_route_skip_tls_verification = false
```

Reply:
`app_skip_tls_verification` might be easier to understand for a user?

Reply:
your right. `route` doesnt make it clearer. 

This setting would be used for custom routes in app plugins where Grafana proxies the request for the plugin. I would prefer to have an configuration for it.

Reply:
Ok, thanks for clarifying.

It looks like there's already per-app configuration, so perhaps this setting would be better configured per-app?

Reply:
One downside to making the setting per-app is you'd need a HTTP transport per app, which probably means more connections since connections will not be re-used between apps.

Reply:
I'll use a global setting that applies to all app plugins.